### ISSUE TITLE
Full screen mode

### ISSUE THREAD

**Issue by adjohnson916** [adjohnson916/magnificent.js#7]

**Comment by adjohnson916** [comment]
Note: An issue with experimental implementation so far&mdash;scale does not maintain proportions when in-page element has different dimensions than screen.
Could be fixed with #13.

**Comment by lamp5** [comment]
Any updates about full screen functionality??

**Comment by AndersDJohnson** [comment]
> Any updates about full screen functionality??

@lamp5 No, I haven't been adding features to this library lately. Would you like to take a stab implementing full screen functionality? I might recommended [screenfull](https://www.npmjs.com/package/screenfull).
